ios/FluentUI/Navigation/BadgeLabelButton.swift

```diff
@@ -208,5 +208,19 @@ class BadgeLabelButton: UIButton {
 
     @objc private func badgeValueDidChange() {
         updateBadgeLabel()
+        updateAccessibilityLabel()
+    }
+
+    private func updateAccessibilityLabel() {
+        guard let item = item else {
+            return
+        }
+        if let itemAccessibilityLabel = item.accessibilityLabel, let badgeAccessibilityLabel = item.badgeAccessibilityLabel {
```

Comment:
Is there a scenario that makes sense to have only the badgeAccessibilityLabel set?
Something like "3 new notifications, button" instead of "Notifications, 3 new items, button".

I'm questioning that because if the accessibilityLabel set to nil will always prevent the badgeAccessibilityLabel from being set.

What are your thoughts on this logic:

```
if let badgeAccessibilityLabel = item.badgeAccessibilityLabel {
  if let itemAccessibilityLabel = item.accessibilityLabel {
    String.localizedStringWithFormat("Accessibility.BadgeLabelButton.LabelFormat".localized,itemAccessibilityLabel,badgeAccessibilityLabel)
  } else {
    accessibilityLabel = badgeAccessibilityLabel
  }
} else {
    accessibilityLabel = item.accessibilityLabel
}
```

Reply:
Yeah!! Makes sense.